core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -114,9 +114,38 @@ private[ui] class ExecutorsPage(
 
   /** Render an HTML row representing an executor */
   private def execRow(info: ExecutorSummary, logsExist: Boolean): Seq[Node] = {
+    // When GCTimePercent is edited change ToolTips.TASK_TIME to match
+    val GCTimePercent = 0.1
+
     val maximumMemory = info.maxMemory
     val memoryUsed = info.memoryUsed
     val diskUsed = info.diskUsed
+
+    // Determine Color Opacity from 0.5-1
+    // activeTasks range from 0 to all cores
+    val activeTasksAlpha =
+      if (info.totalCores > 0) {
+        (info.activeTasks.toDouble / info.totalCores) * 0.5 + 0.5
```

Comment:
So this isn't taking into account that a task could use more then one core via the spark.task.cpus configuration.